hw/dv/tools/xcelium/cov_report.tcl

```diff
@@ -0,0 +1,31 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+# Generate reports for the merged coverage in HTML and text format.
+
+# Set the input merged coverage database directory using the env var 'cov_merge_db_dir'.
+# The supplied env var may have quotes or spaces that needs to be trimmed.
+set cov_merge_db_dir [string trim $::env(cov_merge_db_dir) " \""]
+
+# Set the output directory for the reports database using the env var 'cov_report_dir'.
+# The supplied env var may have quotes or spaces that needs to be trimmed.
+set cov_report_dir [string trim $::env(cov_report_dir) " \""]
+
+# Set the DUT name.
+set dut [string trim $::env(DUT_TOP)]
+set dut_uc [string toupper $dut]
+set dut_instance [string trim $::env(dut_instance)]
+
+# Load the merged coverage database.
+load -run $cov_merge_db_dir
+
+# Generate the text reports (summary is sufficient).
+report -out $cov_report_dir/cov_report.txt -summary -inst uvm_pkg $dut  \
```

Comment:
@rasmus-madsen 
I am using `report` cmd for the text report and `report_metrics` for html. I get a warning that `report` will be deprecated in future releases of IMC. Since the one we have is the latest, this may cause a problem.

Reply:
I missed that I think you should use report_metrics
should give you he same - just add 
-report_type html
you probably also want the -overwrite